lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1753,11 +1785,9 @@ class HybridTextEditing {
   ///
   /// The constructor also decides which text editing strategy to use depending
   /// on the operating system and browser engine.
-  HybridTextEditing() {
-    channel = TextEditingChannel(this);
-  }
+  HybridTextEditing();
 
-  late TextEditingChannel channel;
+  late final TextEditingChannel channel = TextEditingChannel(this);
```

Comment:
Does this still need to be `late`?

Reply:
I think `late` is still required because the right-hand-side expression refers to `this`.

Reply:
Yes: https://github.com/flutter/engine/pull/28333#discussion_r701494663